apps/predictions/test/repo_test.exs

```diff
@@ -37,7 +37,7 @@ defmodule Predictions.RepoTest do
 
     test "filters by min_time" do
       min_time = Util.now() |> Timex.shift(minutes: 15)
-      predictions = Repo.all(route: "Orange", min_time: min_time)
+      predictions = Repo.all(route: "39", min_time: min_time)
```

Comment:
Was this changed because there were no predictions for orange line or?

Reply:
@meagonqz yes, although I just did try changing it back to `Orange` and now it works fine...going to leave it on `39`.